package.json

```diff
@@ -281,7 +281,7 @@
     "@types/has-ansi": "^3.0.0",
```

Comment:
__question:__ how about updating `babel-jest` to `24.1.0` here and in `x-pack`?

Reply:
`babel-jest@v24` uses `babel@v7` which require `@kbn/babel-preset/node_preset` update. seems like quite a bit task

Reply:
will be done in https://github.com/elastic/kibana/pull/32326